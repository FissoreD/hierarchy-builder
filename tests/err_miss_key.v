From HB Require Import structures.

Fail #[key="Tmiss"]
HB.mixin Record Foo T := {}.