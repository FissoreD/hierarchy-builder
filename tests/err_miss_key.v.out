The command has indeed failed with message:
The #[key="Tmiss"] attribute does not match the selected subject T
